spacy/schemas.py

```diff
@@ -209,6 +210,8 @@ class TokenPatternOperator(str, Enum):
 UnderscoreValue = Union[
     TokenPatternString, TokenPatternNumber, str, int, float, list, bool
 ]
+IobString = Literal["", "I", "O", "B"]
+IobStringValue = Union[TokenPatternString, IobString]
```

Comment:
Looking at this again, since the conversion isn't recursive and `IN` etc. aren't going to work, I think it would be better to just support a limited use of `ENT_IOB`:

```suggestion
IobValue = Literal["", "I", "O", "B", 0, 1, 2, 3]
```

I don't think this is all that limiting given that there are only three possible values anyway.

Reply:
OK, sure. I was not sure about numbers though, I thought users most probably wouldn't wanna match with number values, letters are more straightforward. Still, I'll add it.